packages/data-context/src/globalContext.ts

```diff
@@ -0,0 +1,49 @@
+import type { DataContext } from './DataContext'
```

Comment:
I created this file to bypass circular dependency.

Reply:
Where were you seeing this at?

Reply:
`pkg/config` uses `getCtx` and `pkg/data-context` uses `pkg/config` functions. It causes circular dependency and breaks Cypress. 

This file could be removed and merged back to `index.ts` after fixing [the problems I mentioned in this comment](https://github.com/cypress-io/cypress/pull/22530#discussion_r927213805).

Reply:
@sainthkh I think this can be removed now with your latest changes!

Reply:
Unfortunately, it can't. [As I wrote in the comment below,](https://github.com/cypress-io/cypress/pull/22530#discussion_r932805254) we need `getCtx` from `pkg/data-context` dependency in tests. [Check this code](https://github.com/cypress-io/cypress/pull/22530/files#r935032311)

I thought about injection dummy functions to the test. But [its return value is used to throw errors](https://github.com/cypress-io/cypress/pull/22530/files#r935034146). So, it can cause false positive when `getFilesByGlob` is changed.

I think the only way to roll back this code is to:

1. solve that [`obj.supportFile` issue](https://github.com/cypress-io/cypress/pull/22530#discussion_r927213805)
2. pass `supportFile` from `pkg/data-context`
3. remove `pkg/data-context` from tests in `pkg/config`
4. remove this `globalContext` file.

Reply:
able to delete now with the latest commits?